src/renderer/components/spotlight.tsx

```diff
@@ -100,7 +99,7 @@ export class Spotlight extends React.Component<
     const spotSuggestions: Array<SpotlightItem> = suggestions
       .filter((s) => !('error' in s))
       .map((s: ValidSuggestion) => ({
-        text: path.basename(s.filePath),
+        text: s.filePath.split(/[/\\]/).pop() || '',
```

Comment:
Just curious, what does `s.filePath` look like that makes `path.basename` no longer work? Not saying this is wrong, just curious about the new vs old type that makes us need to Regex split it

Reply:
`path.basename` just isn't available anymore since this is in the renderer process so this is an approximation 😅

Reply:
Ah gotcha, makes good sense! 🙂